site/README.md

```diff
@@ -52,7 +52,7 @@ To see changes in Javadoc, run:
 After site changes are committed, you can publish the site with this command:
 
 ```
-mkdocs gh-deploy
+dev/deploy-site.sh
```

Comment:
should this be `./dev/deploy-site.sh`?

Reply:
@jackye1995  `dev/deploy-site.sh` and `./dev/deploy-site.sh` seem equivalent. Did you mean `../dev/deploy-site.sh`?

Reply:
I do believe they're equivalent, but I also usually see commands referenced with an initial `./` when there's a directory in it.

Reply:
Given the script works anywhere, I think we can instead just say "you can publish the site by executing the script `dev/deploy-site.sh`"